File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/check_in_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class CheckInVO:
    """签到结果"""
    created_at: str = None
    updated_at: str = None
    success: bool = None
    check_in_date: str = None
    consecutive_days: int = None
    points: int = None
    extra_rewards: Dict[str, Any] = None
    checked_in_today: bool = None
    message: str = None

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/CheckInVO.kt
package com.sdkwork.app

data class CheckInVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val success: Boolean? = null,
    val checkInDate: String? = null,
    val consecutiveDays: Int? = null,
    val points: Int? = null,
    val extraRewards: Map<String, Any>? = null,
    val checkedInToday: Boolean? = null,
    val message: String? = null
)
